### DISCUSSION TITLE
A F1 Strategy Dashboard

### DISCUSSION THREAD

**Discussion by Casper-Guo** [theOehrly/Fast-F1#610] [Show and tell]
Most f1 dashboards online right now are focused on telemetry and there isn't a great place to look at strategies. So I threw together something at [armchair-strategist.dev](armchair-strategist.dev).

Bug report highly appreciated! Please open an issue in my [repo](https://github.com/Casper-Guo/F1-Data-Visualization).

**Comment by theOehrly** [comment]
Nice, looks great :) I haven't had time to play with it much but so far it worked well.
Your link above isn't working properly though.

**Reply by Casper-Guo** [reply]
I will have to look into that. I migrated server yesterday because AWS free tier doesn’t have enough RAM and have been having DNS issues ever since

**Reply by theOehrly** [reply]
No, it's just a Github/Markdown thing. The URL is fine, but it doesn't go to that URL when you click on it. It goes to some Github page that doesn't exist.

**Comment by ruilov** [comment]
beautiful project
